src/contact.ts

```diff
@@ -107,20 +110,103 @@ export class Contact implements Sayable {
     }
   }
 
+  /**
+   * Get the weixin number from a contact
+   * Sometimes cannot get weixin number due to weixin security mechanism, not recommend.
+   * @returns {string | ''}
```

Comment:
@returns {string}

or {string|null} is better, for null is standing for we can not get the wexin id.